python/ray/tests/test_advanced_3.py

```diff
@@ -128,6 +128,9 @@ def test_global_state_api(shutdown_only):
 
     assert ray.objects() == {}
 
+    ray.put(1)
```

Comment:
@rkooo567 I added a test here just to make sure it could acquire items back

Reply:
That's awesome!